src/trace_link/kineto_operator.py

```diff
@@ -25,6 +25,7 @@ class KinetoOperator:
         stream (Optional[int]): CUDA stream identifier associated with the operator.
         rf_id (Optional[int]): Record function identifier.
         correlation (int): Identifier used to correlate CUDA runtime and GPU operations.
+        pg_name (str): Process Group name for the collective communication.
```

Comment:
pg_name is defined as Optional[str] according to line number 55. Please update the docstring accordingly.

Reply:
This should be Optional[str]. 
I will update this.